web/slide_puzzle/lib/src/core/puzzle_proxy.dart

```diff
@@ -0,0 +1,21 @@
+import 'dart:math' show Point;
```

Comment:
Copyright header on this and the other new files?

Reply:
Hmm... Do we need copyright headers if we aren't the original author?

Reply:
What was the original source for the slide puzzle? Was this something @yjbanov or @kevmoo did?

Reply:
https://github.com/kevmoo/slide_puzzle

Reply:
In that case, for consistency I would go ahead and add headers for consistency with The Flutter Team or The Dart Project listed as the author.